### DISCUSSION TITLE
How to deal with multimodality in TextGrad

### DISCUSSION THREAD

**Discussion by felixgao** [zou-group/textgrad#36] [General]
Hi, 

I have many different documents that is in JPG, PNG, or PDF format that I need to extract certain information from them.  How do I use TextGrad to optimize the LLMs with multimodality? 

Thanks

**Comment by mertyg** [comment]
Currently, TextGrad only supports text.
However, we’ll have news on this soon! :)

**Comment by mertyg** [comment]
It's [here](https://github.com/zou-group/textgrad/blob/main/examples/notebooks/Tutorial-MultiModal.ipynb) now! We'll soon push some benchmark results too. :)
